lib/src/vector_math/plane.dart

```diff
@@ -6,7 +6,7 @@ part of vector_math;
 
 class Plane {
   final Vector3 _normal;
-  double _constant;
+  double constant;
```

Comment:
@rakudrama – FYI – I turned a few things that were properties into fields – assuming that's okay